API/models/Posts.js

```diff
@@ -0,0 +1,177 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+// TODO: implement comments model
+// require('./Comments');
+// const Comment = mongoose.model('Comment');
+
+const PostSchema = new Schema({
+	author: {
+    //type: Schema.Types.ObjectId,
+    type: String,
+    required: true,
+    ref: 'User'
+  },
+  subscribedUsers: {
+    type: Array,
+    default: [],
+    ref: 'User'
+  },
+	content: {
+		type: String,
+		required: true,
+  },
+  image: {
+    type: Buffer,
+  },
+	tags: {
+		type: Array,
+		required: true,
+  },
+  comments: {
+    type: Array,
+    ref: 'Comment'
+  },
+  createdAt: {
+    type: Object,
+  },
+  updatedAt: {
+    type: Object,
+  }
+});
+
+PostSchema.statics.create = function(post) {
+  // Implement a character limit client-side
```

Comment:
We shouldn't be making any assumptions about the client when developing the API, but we can always revisit this